README.md

```diff
@@ -37,6 +37,14 @@ to your Github account:
 
 You should now receive an email whenever the changelog is updated.
 
+---
+### 16 June 2023
+The following changes were deployed into sandbox:
+
+#### individuals-reliefs-api
+
+* For version 1.0 and later, Gov-Test-Scenario values that are not supported by the sandbox will now result in a status code 400 response with an error code of RULE_INCORRECT_GOV_TEST_SCENARIO.
```

Comment:
Is it right to say for v1.0 and later? As there is only v1.0.

Reply:
I also thought about this, but if in the future there is a v2, the change would apply to that. But I suppose since there is no later version at this moment, it would make more sense to have v1.0

Reply:
Yeah I think the changelog should be a historical log of what has been done at that time, so I think as only v1.0 has been updated at this time it wouldn't make sense to say `and later`.